packages/flutter/test/widgets/text_selection_test.dart

```diff
@@ -1244,7 +1244,7 @@ void main() {
       expect(endDragUpdateDetails!.globalPosition, newLocation);
 
       await gesture2.up();
-      await tester.pump(const Duration(milliseconds: 20));
+      await tester.pump(const Duration(milliseconds: 300));
```

Comment:
What happens if you don't make this change?

Reply:
The 300ms android timer doesn't get dismissed and test fails due to remaining timer. I thought canceling it in `dispose` should fix this but doesn't work here even though it fixed other test failures. Any ideas?